src/main/java/com/farao_community/farao/gridcapa/job_launcher/JobLauncherAutoService.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (c) 2021, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.farao_community.farao.gridcapa.job_launcher;
+
+import com.farao_community.farao.gridcapa.task_manager.api.TaskDto;
+import com.farao_community.farao.gridcapa.task_manager.api.TaskStatus;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.cloud.stream.function.StreamBridge;
+import org.springframework.context.annotation.Bean;
+import org.springframework.stereotype.Service;
+import reactor.core.publisher.Flux;
+
+import java.util.Objects;
+import java.util.function.Consumer;
+
+/**
+ * @author Amira Kahya {@literal <amira.kahya at rte-france.com>}
+ */
+@Service
+@ConditionalOnProperty(name = "job-launcher.auto", havingValue = "true")
+public class JobLauncherAutoService {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(JobLauncherAutoService.class);
+    private static final String RUN_BINDING = "launch-task-auto";
```

Comment:
This should match the binding in the configuration. So either define "launch-task-auto" in the application.yml or use an already existing one like "run-task" or "run-task-auto". I think you can use the same, I can't see eny use case where you would need both automatic mode and scheduled mode working together and sending messages to different exchange. SO I think you can directly set here RUN_BINDING="run-task-auto".